kvdb/postgres/db_test.go

```diff
@@ -0,0 +1,199 @@
+package postgres
```

Comment:
would be nice to also run the `walletdb` test suite (github.com/btcsuite/btcwallet/walletdb/walletdbtest)

Reply:
Ah, that is interesting. Didn't know that that existed.

Reply:
Made change to run the suite on postgres.